packages/webapp/src/config.ts

```diff
@@ -10,6 +10,8 @@ export const PROVIDER_NAME = process.env.REACT_APP_PROVIDER_NAME!;
 export const FOMO_WEBSOCKET = process.env.REACT_APP_WEB_SOCKET!;
 export const PROVIDER_KEY = process.env.REACT_APP_PROVIDER_KEY!;
 export const ETHERSCAN_API_KEY = process.env.REACT_APP_ETHERSCAN_API_KEY!;
+export const RECAPTCHA_KEY = process.env.REACT_APP_RECAPTCHA_KEY!;
+export const RECAPTCHA_ACTION_NAME = process.env.REACT_APP_RECAPTCHA_ACTION_NAME!;
```

Comment:
Move this out of environment and into package vars

Reply:
Not sure I get it, you mean to use the hardcoded strings in the codebase? 
`RECAPTCHA_KEY` is a key that will get changed based on environment, so it should be there.
`RECAPTCHA_ACTION_NAME` - it's beneficial from a security standpoint to have this variable in "harder" access. It doesn't really "make or break" it, but small things compound.